src/renderer/screens/lend/History/index.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import React, { useMemo } from "react";
+import { useTranslation } from "react-i18next";
+import type { AccountLikeArray, Operation } from "@ledgerhq/live-common/lib/types";
+import Box from "~/renderer/components/Box";
+import OperationsList from "~/renderer/components/OperationsList";
+import EmptyState from "../EmptyState";
+
+type Props = {
+  navigateToCompoundDashboard: () => void,
+  accounts: AccountLikeArray,
+};
+
+const useCompoundHistory = (accounts: AccountLikeArray): AccountLikeArray => {
+  const filterOps = (op: Operation): boolean => ["REDEEM", "SUPPLY"].includes(op.type);
+  const history = useMemo(
+    () =>
+      accounts.map(acc => {
+        const operations = acc.operations.concat(acc.pendingOperations).filter(filterOps);
```

Comment:
we shouldn't do this. pendingOperations and operations will get duplicated for 30mn.

this would deserve a helper on live-common too, maybe it is actually an evolution of the "groupPerDay" to provide a filter function. because i'm not super fan of copying the account do just do this.

so I would suggest that you give to OperationsList a filter function, that under the hood would get passed to live-common's logic

Reply:
waiting for https://github.com/LedgerHQ/ledger-live-common/pull/912 to finalize this